lib/digit_match.rb

```diff
@@ -0,0 +1,5 @@
+# Returns count of digits matching in the two input non-negative integers
+def digit_match(number_1, number_2)
+  puts "NOT IMPLEMENTED"
+  return -1
```

Comment:
How would you feel about doing `raise NotImplementedError` here instead?

Best practice I think is generally to avoid `puts` statements and other things that will print to the console for code being tested, because that output kind of gets randomly interspersed with the test runner's output.

Reply:
Also this is a very minor quibble, but something irks me about the variable names `number_1` and `number_2`. Perhaps we could do `number1` and `number2` or even `left` and `right`.

Reply:
Both now corrected.